packages/radio-group/test/lion-radio-group.test.js

```diff
@@ -1,144 +1,157 @@
-import { expect, fixture, nextFrame, html } from '@open-wc/testing';
-import { Required } from '@lion/validate';
 import '@lion/radio/lion-radio.js';
-
+import { Required } from '@lion/validate';
+import { expect, fixture, html, nextFrame } from '@open-wc/testing';
 import '../lion-radio-group.js';
 
 describe('<lion-radio-group>', () => {
-  it('has a single checkedValue representing the currently checked radio value', async () => {
+  it('has a single modelValue representing the currently checked radio value', async () => {
     const el = await fixture(html`
-      <lion-radio-group>
-        <lion-radio name="gender[]" .choiceValue=${'male'}></lion-radio>
-        <lion-radio name="gender[]" .choiceValue=${'female'} checked></lion-radio>
-        <lion-radio name="gender[]" .choiceValue=${'alien'}></lion-radio>
+      <lion-radio-group name="gender">
+        <lion-radio .choiceValue=${'male'}></lion-radio>
+        <lion-radio .choiceValue=${'female'} checked></lion-radio>
+        <lion-radio .choiceValue=${'other'}></lion-radio>
       </lion-radio-group>
     `);
     await nextFrame();
-    expect(el.checkedValue).to.equal('female');
+    expect(el.modelValue).to.equal('female');
     el.formElementsArray[0].checked = true;
-    expect(el.checkedValue).to.equal('male');
+    expect(el.modelValue).to.equal('male');
     el.formElementsArray[2].checked = true;
-    expect(el.checkedValue).to.equal('alien');
+    expect(el.modelValue).to.equal('other');
   });
 
-  it('can set initial checkedValue on creation', async () => {
-    const checkedValue = 'alien';
+  it('will register child elements with the same name', async () => {
     const el = await fixture(html`
-      <lion-radio-group .checkedValue="${checkedValue}">
-        <lion-radio name="gender[]" .choiceValue=${'male'}></lion-radio>
-        <lion-radio name="gender[]" .choiceValue=${'female'}></lion-radio>
-        <lion-radio name="gender[]" .choiceValue=${'alien'}></lion-radio>
+      <lion-radio-group name="gender">
+        <lion-radio .choiceValue=${'male'}></lion-radio>
+        <lion-radio .choiceValue=${'female'} checked></lion-radio>
+        <lion-radio .choiceValue=${'other'}></lion-radio>
       </lion-radio-group>
     `);
-
     await nextFrame();
-    await el.registrationReady;
-    await el.updateComplete;
-
-    expect(el.checkedValue).to.equal('alien');
-    expect(el.formElementsArray[2].checked).to.be.true;
+    expect(el.formElementsArray[0].name).to.equal('gender');
+    expect(el.formElementsArray[1].name).to.equal('gender');
+    expect(el.formElementsArray[2].name).to.equal('gender');
   });
 
-  it('can handle complex data via choiceValue', async () => {
```

Comment:
Add it back or explain why you delete

Reply:
Ah, I see now it was moved down the file for organisation purposes but git did not catch the change properly